style/checkstyle.xml

```diff
@@ -18,29 +18,47 @@ specific language governing permissions and limitations
 under the License.
 -->
 <!DOCTYPE module PUBLIC
-    "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
-    "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+        "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
 
 <module name="Checker">
-	<module name="SuppressionFilter">
-    <property name="file" value="style/checkstyle_suppressions.xml"/>
-	</module>
+    <module name="SuppressionFilter">
+        <property name="file" value="style/checkstyle_suppressions.xml"/>
+    </module>
 
-	<module name="Header">
-	    <property name="headerFile" value="style/licence_header.txt"/>
-	    <property name="fileExtensions" value="java"/>
-	</module>
+    <module name="Header">
+        <property name="headerFile" value="style/licence_header.txt"/>
+        <property name="fileExtensions" value="java"/>
+    </module>
+    
+    <module name="FileTabCharacter">
```

Comment:
The contribute.md states that spaces are used instead of tabs.
A lot of files do not stick to this convention.